### ISSUE TITLE
How to store a set of arbitrary values and pass it to Lua?

### ISSUE THREAD

**Issue by TheBoxyBear** [NLua/NLua#450]
I have a class that I want to support custom data defined in Lua in order to make the type more flexible for the user, however I'm struggling to find the right C# to use to store the dataset. I tried using dynamic, Dictionary<string, object> and LuaTable, but every time I get the error: `[string "?"]:1: global 'getmetatable' is not callable (a nil value)`.

How would I go about storing custom data and passsing it to the interpreter?

**Comment by viniciusjarina** [comment]
I am not sure what you trying to do. But you can pass  any C# `object` to Lua context using the `[]`

**Comment by TheBoxyBear** [comment]
Yes, I'm able to pass objects of pre-defined types, however the data I'm trying to pass is dynamic, with member names/keys defined at runtime in Lua.

**Comment by taemyr** [comment]
Do you have some form of sandbox set up?

Could you show the lua script that throws the error?
